lib/src/settings/embedder.dart

```diff
@@ -0,0 +1,262 @@
+import './distribution.dart';
+
+sealed class Embedder {
+  final String source;
+
+  const Embedder({
+    required this.source,
+  });
+
+  Map<String, Object?> toMap();
```

Comment:
since `source` is shared between all embedders, this can be changed to:
```suggestion
  Map<String, Object?> toMap() {
    'source': source,
  }
```

Reply:
I've updated the code, now it is mapped only once in the base class as you described